mohammad/JS/JS-simpleDataFetching/main.js

```diff
@@ -0,0 +1,11 @@
+const titleArr = new Array();
+const bodyArr = new Array();
+for (let i = 1; i <= 10; i++) {
+  fetch(`https://jsonplaceholder.typicode.com/posts/${i}`)
+    .then((response) => response.json())
+    .then((post) => {
+      titleArr.push(post["title"]);
+      bodyArr.push(post["body"]);
+      if (i == 10) console.table([titleArr, bodyArr]);
```

Comment:
This method is not suitable for output because the requests are executed asynchronously, and there is no guarantee that when a post with ID 10 succeeds, the previous requests have been successful or not. As a result, the number of posts in the output is different each time.
Try to use Promise methods